safe/test/data/gisv4/hazard/tsunami_wgs84.xml

```diff
@@ -0,0 +1,332 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you make a copy of this layer ? I think you should use safe/test/data/hazard, no ?